tests/run_tests.sh

```diff
@@ -79,4 +81,11 @@ if [[ -n "${CURATION_TESTS_FILEPATH}" ]]; then
     "${CURATION_TESTS_FILEPATH}")
 fi
 
+if [[ -n "${TEST_PATTERN}" ]]; then
```

Comment:
This I don't completely understand. is this for just adding a --test_pattern param?

Reply:
Yea, I like to be able to run a specific unit/integration test from the command line without being required to know the file path.  `TEST_PATTERN` lets me do that.  It's already built into the runner file and was part of the previous bash script.  It just got refactored away.  This functions just like the `CURATION_TESTS_FILEPATH` env variable.  I would prefer just to pass a command line parameter instead of setting an ENV var, but that would have been a more significant change than I wanted to make at this time.

Reply:
I used it to run `TEST_PATTERN=admin_api* ./tests/run_tests.sh unit` so I could just run the one class.